src/uwtools/tests/test_cli.py

```diff
@@ -165,6 +150,23 @@ def test__check_template_render_vals_args_noop_explicit_valsfmt():
     assert cli._check_template_render_vals_args(args) == args
 
 
+def test__check_verbosity_fail(capsys):
+    log.setLevel(logging.INFO)
+    args = {STR.quiet: True, STR.verbose: True}
+    with raises(SystemExit):
+        cli._check_verbosity(args)
+    assert (
+        "Specify at most one of %s, %s, or %s"
+        % (cli._switch(STR.debug), cli._switch(STR.quiet), cli._switch(STR.verbose))
+        in capsys.readouterr().err
+    )
+
+
+def test__check_verbosity_ok():
+    args = {"foo": 88}
+    assert cli._check_verbosity(args) == args
```

Comment:
I think this could also be parametrized to check the combinations we care about (and apologies for the initial not-great test, which I probably wrote...), maybe something like
```
@pytest.mark.parametrize("flags", ([STR.debug], [STR.quiet], [STR.verbose], [STR.debug, STR.verbose]))
def test__check_verbosity_ok(flags):
    args = {flag: True for flag in flags}
    assert cli._check_verbosity(args) == args
```

i.e. It's ok to use each flag on its own, it's ok to use `--debug` and `--verbose` together, but it's not ok to use `--quiet` with anything else.

Reply:
Ah thats true, and that may fix the later test too